roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -43,13 +54,13 @@ def tags(self):
         """
         return []
 
-    @classmethod
-    def is_active(cls, task_vars):  # pylint: disable=unused-argument
+    @staticmethod
```

Comment:
It sucks when the tools get in the way :-D
Makes us invent more hacks to workaround the tools:

```python
def is_active(self):
    "Some docstring"
    return self
```

or

```python
def is_active(self):
    "Some docstring"
    return self and True
```

(both aberrations that would probably "make the tool happy")